launcher/main.ts

```diff
@@ -150,8 +150,8 @@ function connect(openServer: OpenServer) {
         `(got app API version ${serverInfo.appApiVersion}, expected ${appApiVersion}).`);
         return;
       }
-
-      openServer.window.loadURL(`http://${openServer.address}`);
+      let protocol = serverInfo.useSSL ? "https" : "http";

```

Comment:
I think this could be `const` rather than `let`, right? We're trying to use `const` whenever possible.